### PR TITLE
Blog post for Spirv

### PR DESCRIPTION
**Pull request by ykafia**
I've decided to try myself at writing a series of blog posts for spir-v!

### CODE REVIEW

File: posts/investigating-spirv-for-the-shader-system.md [hunk 1]
```diff
@@ -1,12 +1,13 @@
 ---
-title: "Investigating spir-v for the shader system"
+title: "Investigating spir-v for the shader system - Part 1"
 author: youness
 popular: false
 image: /images/spir.png
 tags: ['.NET', 'spir-v', 'shader', 'hlsl', 'glsl', 'sdsl', 'compilation']
```
Comment (VaclavElias) [inline posts/investigating-spirv-for-the-shader-system.md:6]: Let's keep just major tags, as these are for a better navigation in this page https://www.stride3d.net/tags/. I would keep: .NET, Shaders.

We can add more tags later, if there will be more blog posts specific/related to some topic.

Shaders maybe, probably as plural, as we have there: Particles, Prefabs,

File: images/blog/spirv/shaders-explanation.png
Comment (VaclavElias) [inline images/blog/spirv/shaders-explanation.png:1]: If we are going to reuse this image (Part 2) let's keep it in the current location. If not, could you move it rather to images/blog/2023-11 so we don't create too many folders for each blog?

File: posts/investigating-spirv-for-the-shader-system.md (hunk 1 above)
Comment (ykafia) [reply posts/investigating-spirv-for-the-shader-system.md:6]: Yes of course!
That makes sense, i'll make the changes!

File: images/blog/spirv/shaders-explanation.png
Comment (ykafia) [reply images/blog/spirv/shaders-explanation.png:1]: I moved it to 2023-11, there's a high chance i won't use it for the following parts!

**Comment by ykafia** [comment]
I'm very happy with the the latest commit i made, if everything is fine for everyone we could merge it to master ?

**Pull request merged**